jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -69,7 +72,8 @@ def process(self, input_table):
             output_model.meta.cal_step.wfs_combine = 'COMPLETE'
             output_model.meta.asn.pool_name = asn_table['asn_pool']
             output_model.meta.asn.table_name = os.path.basename(input_table)
-            output_model.meta.filename = which_set['name']
+            # format the filename here
+            output_model.meta.filename = which_set['name'].format(suffix=self.suffix) + ".fits"
```

Comment:
We normally don't like to hardwire the file type (.fits in this case), so that models could be stored in different formats, if desired. But this is such a corner case, I guess it's OK in this instance.